Makefile

```diff
@@ -34,7 +34,7 @@ test:
 	cargo test --all --all-features
 	# Compile the code generated through tests.
 	cd tests/test_pet && cargo check
-	cd tests/test_pet/cli && CARGO_TARGET_DIR=../target cargo check
+	cd tests/test_pet/cli && CARGO_TARGET_DIR=../target cargo build
```

Comment:
Checking ensures that the code compiles and doesn't build anything unless required. We don't want the `test_pet` CLI because we're already building the huge `test_k8s` CLI (which I'd really like to get rid of, but not yet) - building binaries consume time and I don't want the CI to take long times.